cmd/collectors/zapi/collector/zapi.go

```diff
@@ -103,6 +113,25 @@ func (me *Zapi) InitVars() error {
 	return nil
 }
 
+func (me *Zapi) LoadPlugin(kind string, abc *plugin.AbstractPlugin) plugin.Plugin {
+	switch kind {
```

Comment:
may be log for default case in switch?

Reply:
done - this  method will eventually be rewritten using modules too